src/lib.rs

```diff
@@ -331,7 +333,7 @@ impl GammaLut {
                 pixel[0] = table_g[luminance as usize];
                 pixel[1] = table_g[luminance as usize];
                 pixel[2] = table_g[luminance as usize];
-                // Don't touch alpha
+                pixel[3] = table_g[luminance as usize];
```

Comment:
(we are trying to figure out what this does and why)

Reply:
This basically sets the alpha channel so that we can use the BGRA image as an alpha mask. In theory we don't need to have the rest of the channels and this function could just return an alpha_mask

Reply:
Yep - we always currently return a BGRA, even for alpha fonts. Without this change, the gamma adjustment does something like (6,6,6,6) -> (30,30,30,6). Since we're (now) using pre-multiplied alpha for text runs, the alpha value must be <= the RGB values to get valid output.